process_report/invoices/billable_invoice.py

```diff
@@ -140,19 +166,19 @@ def get_initial_credit_amount(
         )
         print(f"New PI Credit set at {new_pi_credit_amount} for {self.invoice_month}")
 
-        current_pi_set = set(data[invoice.PI_FIELD])
+        credit_eligible_projects = self._get_credit_eligible_projects(data)
```

Comment:
@naved001 I think I'll make an issue in the near future to replace every `print()` statement with logging. @knikolla @naved001 I will use your judgement on when that "near future" is.

Reply:
I don't think it's necessary to log this for now, since the credit eligible projects will have a credit code associated to them (unless they have no credit).